apiserver/utility/utility.go

```diff
@@ -53,18 +59,25 @@ func ValidateBitriseConfigAndSecret(bitriseConfig, secretsConfig string) error {
 	}
 
 	if len(errorStrs) > 0 {
-		return fmt.Errorf("Validation failed: %s", strings.Join(errorStrs, " | "))
+		return nil, fmt.Errorf("Validation failed: %s", strings.Join(errorStrs, " | "))
 	}
 
 	if len(validationOutput.Error) > 0 {
-		return fmt.Errorf("bitrise validation command failed, error: %s", validationOutput.Error)
+		return nil, fmt.Errorf("bitrise validation command failed, error: %s", validationOutput.Error)
 	}
 
 	if cmdErr != nil {
-		return fmt.Errorf("bitrise validation command failed, error: %s", cmdErr)
+		return nil, fmt.Errorf("bitrise validation command failed, error: %s", cmdErr)
+	}
+
+	warningsArray := []string{}
+	warnings := validationOutput.Data.Config.Warnings
+	warnings = append(warnings, validationOutput.Data.Secrets.Warnings...)
```

Comment:
So we are putting both secret and config related warnings into one slice? I would consider using a more structured format at that point (eg. by creating a custom type), the utility shouldn't be aware how others will use its output, there can be a use case of being able to differentiate between secret and config warnings. (I'm not sure if our use case is one of those, for example.)